docs/zh_CN/ch1/exchangis_deploy_cn.md

```diff
@@ -6,33 +6,34 @@ Exchangis 的安装，主要分为以下四步：
 2. Exchangis 安装部署
 3. DSS ExchangisAppConn 安装部署
 4. Linkis Sqoop 引擎安装部署
+5. Linkis DataX 引擎安装部署
 
 ## 1. Exchangis 依赖环境准备
 
 #### 1.1 基础软件安装
 
 | 依赖的组件                                 | 是否必装 | 安装直通车 |
 |---------------------------------------| ------ | --------------- |
-| MySQL (5.5+)                          | 必装  | [如何安装mysql](https://www.runoob.com/mysql/mysql-install.html) |
-| JDK (1.8.0_141)                       | 必装 | [如何安装JDK](https://www.runoob.com/java/java-environment-setup.html) |
-| Hadoop(2.7.2，Hadoop 其他版本需自行编译 Linkis) | 必装 | [Hadoop单机部署](https://hadoop.apache.org/releases.html) ；[Hadoop分布式部署](https://hadoop.apache.org/releases.html) |
-| Hive(2.3.3，Hive 其他版本需自行编译 Linkis)     | 必装 | [Hive快速安装](https://hive.apache.org/downloads.html) |
+| JDK (1.8.0_141)           | 必装  | [如何安装JDK](https://www.oracle.com/java/technologies/downloads/) |
+| MySQL (5.5+)           | 必装 | [如何安装mysql](https://www.runoob.com/mysql/mysql-install.html) |
+| Hadoop(3.3.4，Hadoop 其他版本需自行编译 Linkis) | 必装 | [Hadoop部署](https://www.apache.org/dyn/closer.cgi/hadoop/common/hadoop-3.3.4/hadoop-3.3.4.tar.gz) |
```

Comment:
hadoop3.3.4 is sure?

Reply:
Hadoop was upgraded to 3.3.4 as with linkis1.4.0.